apps/cookbook/src/app/guides/chart-config-guide/chart-config-guide.component.html

```diff
@@ -0,0 +1,129 @@
+<h1>Chart config</h1>
+<p>
+  The Kirby charts are based on a config strategy. The aim of this is to give the implementer full
+  control of the chart. Kirby simply provides some configurations which can be injected into a
```

Comment:
```suggestion
  control of the chart. Kirby simply provides some configurations which can be applied to a
```
Consider this to not cause confusion around dependency injection or similar concepts.